logicrunner/outgoingsender.go

```diff
@@ -109,9 +114,19 @@ func (rs *outgoingRequestSender) SendAbandonedOutgoingRequest(ctx context.Contex
 	}
 }
 
-func (rs *outgoingRequestSender) Stop(_ context.Context) {
-	// not implemented yet, see INS-3293, INS-3371
-	// rs.as.CloseAll()
+func (rs *outgoingRequestSender) Stop(ctx context.Context) {
+	resultChan := make(chan struct{}, 1)
+	msg := stopOutgoingRequestSenderMessage{
+		resultChan: resultChan,
+	}
+	err := rs.as.SendPriority(rs.senderPid, msg)
+	if err != nil { // this is unlikely to happen
```

Comment:
for unlikely use panic to be able to backtrace

Reply:
I don't think I agree. IIRC there is no such rule in our code style. Also the only scenario when SendPriority can fail is when actor we send a message to doesn't exist.